src/views/Pipeline.vue

```diff
@@ -359,7 +361,23 @@ export default defineComponent({
       getPinnedJobs: 'user/getPinnedJobs'
     })
   },
-  methods: {
+  methods : {
+    isSelected(JobEnumId: any) {
+    return (this as any).fEnumId.some((jobId: any) =>  jobId === JobEnumId );  
+    },
+    updateFilteredEnum(JobEnumId: any) {
+      
+      const isAlreadyAdded = this.isSelected(JobEnumId);
```

Comment:
Get indexOf, use it to check if it is already in list and then perform operation

Reply:
Improved sir